python/sglang/srt/server_args.py

```diff
@@ -223,20 +223,23 @@ def __post_init__(self):
 
         # Set mem fraction static, which depends on the tensor parallelism size
         if self.mem_fraction_static is None:
-            if gpu_mem <= 81920:
-                if self.tp_size >= 16:
-                    self.mem_fraction_static = 0.79
-                elif self.tp_size >= 8:
-                    self.mem_fraction_static = 0.81
-                elif self.tp_size >= 4:
-                    self.mem_fraction_static = 0.85
-                elif self.tp_size >= 2:
-                    self.mem_fraction_static = 0.87
-                else:
-                    self.mem_fraction_static = 0.88
+            if self.tp_size >= 16:
+                self.mem_fraction_static = 0.79
+            elif self.tp_size >= 8:
+                self.mem_fraction_static = 0.81
+            elif self.tp_size >= 4:
+                self.mem_fraction_static = 0.85
+            elif self.tp_size >= 2:
+                self.mem_fraction_static = 0.87
             else:
+                self.mem_fraction_static = 0.88
+            if gpu_mem > 81920:
```

Comment:
Can we use `self.dp_size` here when it's greater than 1?

Reply:
Consider both tp_size and dp_size.

Reply:
Found another critical cause in cuda graph capture bs. This change is reverted.